src/FsAutoComplete.Core/AbstractClassStubGenerator.fs

```diff
@@ -84,77 +98,62 @@ let private tryFindAbstractClassExprInParsedInput
 let tryFindAbstractClassExprInBufferAtPos
   (codeGenService: ICodeGenerationService)
   (pos: Position)
-  (document: Document)
+  (document: NamedText)
```

Comment:
This `document` structure is just a less-well-used NamedText - we should standardize on the new hotness.